tests/test_fdb_update.py

```diff
@@ -146,8 +145,7 @@ def test_FDBAddedAndUpdated(self, dvs, testlog):
         ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
                                             [("mac", "52:54:00:25:06:E9"), ("bvid", bvid)],
                                             [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
-                                             ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet4"]),
-                                             ('SAI_FDB_ENTRY_ATTR_PACKET_ACTION', 'SAI_PACKET_ACTION_FORWARD')])
```

Comment:
why is packet action removed?

Reply:
It was removed since packet action as forward is the default action.